hedera-node/src/test/java/com/hedera/services/store/models/OwnershipTrackerTest.java

```diff
@@ -77,4 +77,19 @@ void newChange(){
 		assertEquals(Id.DEFAULT, change.getNewOwner());
 		assertEquals(treasury, change.getPreviousOwner());
 	}
+
+	@Test
+	void compareChanges() {
+		final var change = new OwnershipTracker.Change(treasury, account, 7L);
+		final var otherChange = new OwnershipTracker.Change(treasury, account, 7L);
+		final var refChange = change;
+
+		boolean result = change.equals(null);
+		assertFalse(result);
+		result = change.equals(treasury);
+		assertFalse(result);
```

Comment:
Note to the reviewers: This is intentional.

https://github.com/hashgraph/hedera-services/pull/1779/commits/44b7f046076040c39ff8c9b96bcfdebe322d9c96